### DISCUSSION TITLE
Question wrt virtual keyboard settings

### DISCUSSION THREAD

**Discussion by giantclambake** [BlitterStudio/amiberry#1191] [Q&A] [answered]
With **one exception** , all the options available in GUI->Virtual Keyboard (which can be saved to the current `.uae` file), can be defined in amiberry.conf as default options ...ie; user always wants vkbd on

The exception is `Use Retroarch Vkbd button` ~ this option can only be saved to the current `.uae` file as;

 amiberry.use_retroarch_vkbd=true

As I'm unsure, I need ask the question ~ is there a case for `default_use_retroarch_vkbd=[yes/no]`in amiberry.conf?

TIA

**Comment by midwan** [comment, marked as answer]
No, that option is not available in `amiberry.conf` - there was no request for it either, so I suspect it's not needed there.

**Reply by giantclambake** [reply]
Thanks ;)
